frontend/packages/dev-console/src/components/import/advanced/ResourceLimitSection.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import FormSection from '../section/FormSection';
+import { ResourceLimitField } from '../../formik-fields';
+import { MemoryUnits, CpuUnits } from '../import-types';
+
+const ResourceLimitSection: React.FC = () => {
+  return (
+    <FormSection title="Resource Limit">
+      <div className="co-section-heading-tertiary">CPU</div>
+      <ResourceLimitField
+        name="limits.cpu.request"
+        unitName="limits.cpu.requestUnit"
+        inputLabel="Request"
+        unitItems={CpuUnits}
+        unitSelectedKey="m"
+        helpText="The maximum amout of CPU the container is allowed to use when running."
```

Comment:
Help text is wrong. Request is not the maximum you can use.

Reply:
@spadgett  Can you please suggest me the correct help text.

Reply:
> The minimum amount of CPU the container is guaranteed.

Reply:
Thanks.